Makefile

```
@@ -0,0 +1,50 @@
+ARTIFACT_ID=k8s-minio
+MAKEFILES_VERSION=8.4.0
+VERSION=2023.9.23-1
```

```suggestion
VERSION=12.8.9-0
```
Maybe start with a "-0", so that the first release gets a "-1"